src/IsaacAppTypes.tsx

```diff
@@ -451,16 +451,6 @@ export interface UserExamPreferences {
 
 export interface SubjectInterests {
     CS_ALEVEL?: boolean;
-    PHYSICS_GCSE?: boolean;
-    PHYSICS_ALEVEL?: boolean;
-    PHYSICS_UNI?: boolean;
-    CHEMISTRY_ALEVEL?: boolean;
-    CHEMISTRY_GCSE?: boolean;
-    CHEMISTRY_UNI?: boolean;
-    MATHS_GCSE?: boolean;
-    MATHS_ALEVEL?: boolean;
-    MATHS_UNI?: boolean;
-    ENGINEERING_UNI?: boolean;
 }
```

Comment:
Another case where we might not need the interface at all?

Reply:
Removed the interface, also removed SUBJECT_INTEREST from user preferences, subjectInterestInput.tsx deleted and also removed CSS "isaac-checkbox" that only appears to have existed there.